packages/gatsby-theme-project-portal/src/layouts/SearchPageLayout.tsx

```diff
@@ -0,0 +1,265 @@
+import React, { FunctionComponent, useEffect, useRef, useState } from "react"
+import lunr from "lunr"
+import { SearchBar } from "../components/SearchBar"
+import { isNull } from "lodash"
+import { graphql } from "gatsby"
+import { ContactType, TopicType } from "../components"
+
+export interface SearchProps {
+  data: {
+    site: {
+      siteMetadata: {
+        siteUrl: string
+      }
+    }
+    allProject: {
+      nodes: {
+        slug: string
+        question: string
+        title: string
+        summary: string
+        status: string
+        opportunityCloses: Date
+        startDate: Date
+        endDate: Date
+        lastModified: Date
+        agency: string
+        topics?: TopicType[]
+        deliverable
+        purpose: string
+        expertise: string
+        requirement: string
+        keyDates: string
+        priorResearch: string
+        statusOfData: string
+        fundingInfo: string
+        emailContent: string
+        mainContact: ContactType
+        projectTeam?: ContactType[]
+        faq: {
+          text: string
+          title: string
+        }[]
+      }
+    }
+    allGeneralPage: {
+      nodes: {
+        slug: string
+        lede: string
+        faq: {
+          text: string
+          title: string
+        }[]
+        aims: {
+          text: string
+          title: string
+        }[]
+        title: string
+      }
```

Comment:
Doesn't this need to be an array too?
```suggestion
      }[]
```